modules/caching-materials-manager-browser/test/caching_materials_manager_browser.test.ts

```diff
@@ -59,4 +59,28 @@ describe('WebCryptoCachingMaterialsManager', () => {
     expect(test._maxBytesEncrypted).to.equal(maxBytesEncrypted)
     expect(test._maxMessagesEncrypted).to.equal(maxMessagesEncrypted)
   })
+
+  it('Precondition: A partition value must exist for WebCryptoCachingMaterialsManager.', () => {
```

Comment:
Does this test also need to be added to caching_materials_manager_node.test.ts ?

Reply:
Yes, and it is in a different PR